lib/charms/lte_core_interface/v0/lte_core_interface.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the `lte-core` relation.
+
+This library contains the Requires and Provides classes for handling the `lte-core`
+interface.
```

Comment:
IMO we could us some more background here; like what was the purpose/motivation behind creation of this interface.

Reply:
Added, let me know what you think.